libexec/rtld-cheri-elf/tests/abi-mismatch/dlopen-hybrid/dlopen-hybrid.c

```diff
@@ -48,16 +48,17 @@ ATF_TC_BODY(dlopen_purecap_fail, tc)
 {
 	char error_msg[PATH_MAX];
 	const char* exedir = get_executable_dir();
-#ifdef __mips__
 	snprintf(error_msg, sizeof(error_msg),
 	    "%s/%s: cannot load %s/../%s since it is CheriABI",
 	    exedir, "dlopen-hybrid", exedir, "libbasic_purecap.so.0");
-#else
-#error "Error message wrong for non-MIPS"
-#endif
 	test_dlopen_failure("libbasic_purecap.so.0", error_msg);
 }
 
+/*
+ * Only MIPS currently defines multiple capability widths for the same base
+ * integer ISA.
+ */
+#ifdef __mips__
```

Comment:
We can probably GC this whole section since we no longer support CHERI256. But that can be done separately.

Reply:
Yes, I was thinking that, and I carefully picked "currently defines" rather than "currently supports" :)